src/tools/interop/idt/capture/ecosystem/play_services/play_services.py

```diff
@@ -53,9 +60,13 @@ def __init__(self, platform: Android, artifact_dir: str) -> None:
                             '168',  # mDNS
                             ]
 
+        if config.test_error_init:
+            self.logger.critical("Throwing exception in init for test!")
+            raise Exception("Test init exception from Play Services!")
```

Comment:
Can you use a derived exception here? That will let you catch it a bit more gracefully if necessary. The base might want to define a general derived exception for the platforms to use.

Reply:
Ditto below

Reply:
Thanks for calling this out, I can do that but it might also be a good idea to remove these, kind of a hacky test.

Reply:
Have removed these temp blocks. Would be nice to write proper tests for this instead.